src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/entities/DraftAdjudication.kt

```diff
@@ -11,6 +11,7 @@ import javax.persistence.Table
 @Table(name = "draft_adjudications")
 data class DraftAdjudication(
   override val id: Long? = null,
+  val reportNumber: Long?,
```

Comment:
This is required so we can filter out drafts that have already been reported

Reply:
Cool